.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -52,24 +51,33 @@ body:
     attributes:
       label: "How to reproduce the issue"
       description: |
-        - Steps to reproduce using `nvim -u NORC` or `nvim -u NONE` (try both).
+        - [Prepare](https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md#reporting-problems) a minimal config that reproduces the problem with `nvim -u minimal.lua` and provide it below.
         - For build failures: list the exact steps including CMake flags (if any).
         - For shell-related problems: try `env -i TERM=ansi-256color "$(which nvim)"`.
       placeholder: |
-        nvim -u NONE
-        :edit foo
-        yiwp
+        1. `nvim -u minimal.lua` 
+        2. `:edit foo` 
+        3. `yiwp` 
     validations:
       required: true
 
   - type: textarea
     attributes:
       label: "Expected behavior"
-      description: "Describe the behavior you expect. May include logs, images, or videos."
+      description: "Describe the behavior you expect. May include logs (`:edit $NVIM_LOG_FILE`), images, or videos."
     validations:
       required: true
   - type: textarea
     attributes:
       label: "Actual behavior"
     validations:
       required: true
+
+  - type: textarea
+    attributes:
+      label: "Minimal config"
+      description: "Minimal(!) configuration necessary to reproduce the issue. You can start with the provided [`minimal.lua`](https://github.com/neovim/neovim/blob/master/contrib/minimal.lua). If _absolutely_ necessary, add plugins and config options from your `init.lua` at the indicated lines."
+      render: Lua
+    validations:
+      required: true
```

Comment:
This is quite some friction to add to bug reports. No objection to having and mentioning the minimal.lua, but making it a required part of bug reports now means that it basically replaces `nvim -u NONE` or `nvim -u NORC` for both bug-reporters and bug-triagers. That is going to be mostly noise for most reports.

Reply:
Here's the basic problem this is supposed to solve: with the advent of `packer.nvim`, vim packages have become a de facto standard for plugins. And neither `-u NONE` nor `-u NORC` is suitable for dealing with those:

1. `NORC` still sources all `start` packages.
2. `NONE` (and `--noplugins`) skip _all_ runtime files -- even the builtin ones.

This has lead to a lot of "bug" reports that are actually conflicts with installed plugins.

This init file is the only way I have found to have a reliably and reproducibly clean state.

Reply:
And, for the record: The reason to have this as a separate field is to

1. not clutter the "steps" further and
2. allow marking this as a code block so you get a) syntax highlighting and b) a button for copying the config.

This reduces friction for maintainers trying to reproduce the issue.

Reply:
It doesn't really reduce friction. I almost never install plugins to repro an issue, and I don't want to start doing so. I'm also imagining a veteran vim user trying to file a bug report and getting hit with these bureaucratic steps, and then just giving us the middle finger...

Reply:
> sounds like we are converging on the _actual_ problem. (does `--clean` help at all, btw?)

No, same issue (`clean` just additionally skips `shada`).